bootstrap-salt.sh

```diff
@@ -3289,7 +3326,25 @@ install_debian_9_git_deps() {
 }
 
 install_debian_10_git_deps() {
-    install_debian_9_git_deps || return 1
+    install_debian_git_pre || return 1
```

Comment:
This is great. Later we could reuse `install_debian_git_pre` in other "git" installation functions for Debian and probably Ubuntu.

Reply:
yep thats what i figured. also to respond to your debian 10 test in drone, i'm working on that in this pr: https://github.com/saltstack/salt-bootstrap/pull/1367  and adding more tests like py3

Reply:
Amesome!

Reply:
Awesome!